charts/sysdig-deploy/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.5
+
+### Bugfixes
+
+* Bumped node-analyzer to 1.5.9
+* Bumped kspm-collector to 0.5.9
```

Comment:
not quite

Reply:
no but this still isn't correct

Reply:
Oops I see you've already mentioned this @lilx1ao

Reply:
yeah. lets hope Mike sees it the THIRD time lol